packages/core-v2-sdk/src/models/MultiPool.ts

```diff
@@ -7,11 +7,21 @@ import { Pool } from "./Pool";
 import { Token } from "./Token";
 import { YieldSource } from "./YieldSource";
 
+/**
+ * MultiPool model class.
+ * @class
+ */
 export class MultiPool {
   address: string;
   context: ElementContext;
   dataSource: MultiPoolDataSource;
 
+  /**
+   * Create a MultiPool model.
+   * @param {string} address - MultiPool contract address
```

Comment:
I didn't catch this in the last review, but I think we can omit the types here. Since we're using TS, The intellisense will already include the types. Idk if they're needed for doc generation, or if there are libraries that will infer the types from the TS, but I figured we could add them in later if we *have* to.

Reply:
Yeah TS will infer the function sig, but using jsdocs will help with doc gen and providing descriptions for the parameters.

Reply:
We can remove them later if we want, just setting a general practice about including jsdoc templates